src/frontend/hooks/use-query-params.ts

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable no-unused-vars */
+import isEmpty from 'lodash/isEmpty.js'
```

Comment:
let's add `src/frontend/hooks/index.js` and export all hooks from it

Reply:
Done